libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/AccountSwitcherActivity.java

```diff
@@ -46,7 +47,7 @@
  *
  * @author bhariharan
  */
-public class AccountSwitcherActivity extends Activity {
+public class AccountSwitcherActivity extends AppCompatActivity {
```

Comment:
I was doing another test and realized `AccountSwitcherActivity` also uses the `SalesforceSDK` style for its theme.  Without being an `AppCompatActivity`, it wouldn't display the AppCompat provided action bar.

This one seems to reparent without any other issues.